packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx

```diff
@@ -1,66 +1,31 @@
-import { useState } from 'react';
 import styled from 'styled-components';
-import { Button, Image, Paragraph } from '@trezor/components';
-import { Translation, CheckItem, Modal } from 'src/components/suite';
+import { Paragraph } from '@trezor/components';
+import { Translation } from 'src/components/suite';
 import { deviceAuthenticityOptOut } from 'src/actions/suite/suiteActions';
 import { useDispatch } from 'src/hooks/suite';
+import { OptOutModalOptOutModal } from './OptOutModal';
 
 const StyledParagraph = styled(Paragraph)`
     text-align: left;
     padding: 16px;
```

Comment:
There should be no additional padding on sides and `spacingsPx` should be used.

Reply:
fixed